README.md

```diff
@@ -1,3 +1,16 @@
+# Adobe User Sync Tool
```

Comment:
It's a pretty sacrosanct Github tradition to tile the readme with the project name.  Is there really a need to violate that here?